This feature allows to omit the exception on a `catch` block.